service/server.py

```diff
@@ -137,6 +131,23 @@ def get_title(title_number):
         abort(404)
 
 
+@app.route('/titles/<title_number>.pdf', methods=['GET'])
+@login_required
+def display_title_pdf(title_number):
```

Comment:
We need to unit test this method:
- calls api_client to get data
- fails if can't get the data
- uses the data to generate the pdf (extracting pdf creator could be useful here)
- returns the generated pdf